error TS7006: Parameter '_meta' implicitly has an 'any' type.

Hello!

Thank you for this great library!

However, when I try to compile it with a simple app, I'm getting the following error:

```
$ tsc

220         const foundMeta = meta.find(_meta => _meta.index === index);
                                        ~~~~~

node_modules/di-typescript/lib/annotations.ts(220,37): error TS7006: Parameter '_meta' implicitly has an 'any' type.
```

```
$ tsc --version
Version 2.2.1
```

**tsconfig.json**

```
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "ES6",
    "alwaysStrict": true,
    "listEmittedFiles": true,
    "noImplicitAny": true,
    "outDir": "dist/",
    "pretty": true,
    "removeComments": true,
    "sourceMap": true,
    "typeRoots": [
      "node_modules/@types"
    ]
  },
  "include": [
    "src/**/*"
  ],
  "exclude": [
    "node_modules/"
  ]
}
```

What could be the problem?

Thanks!

--- a/lib/annotations.ts
+++ b/lib/annotations.ts
@@ -217,7 +217,7 @@ export function Inject(target: any): void {
       const [meta, decorator] = current;
 
       if (meta) {
-        const foundMeta = meta.find(_meta => _meta.index === index);
+        const foundMeta = meta.find((_meta: any) => _meta.index === index);
 
         if (foundMeta) {
           return new decorator(foundMeta.token);

